### ISSUE TITLE
vtkDataSetAttributes not exporting all methods

### ISSUE THREAD

**Issue by WesleyTheGeolien** [Kitware/vtk-js#2567]
### High-level description
Functions from FieldData are not appearing on the vtkDataSetAttributes

### Steps to reproduce
- Use intellisense or run `npx tsc -b`
- create some polyData
- `polyData.getPointData().getArrays()` 
```bash
error TS2339: Property 'getArrays' does not exist on type 'vtkDataSetAttributes'.
```

### Detailed behavior
Looking through my local node modules `node_modules/@kitware/vtk.js/Common/DataModel/DataSetAttributes.d.ts` line 2 `import { IFieldDataInitialValues, vtkFieldData } from '././FieldData';` however FieldData should be at `./DataSetAttributes/FieldData` so I think there is a problem in the handling during the build phase? Looking at: https://github.com/Kitware/vtk-js/blob/master/Sources/Common/DataModel/DataSetAttributes/index.d.ts it appears to be correct.

### Expected behavior
No typescript error

### Environment
- **vtk.js**: @kitware/vtk.js 24.3.1
- **OS**: Mac OSX 12.5

**Comment by WesleyTheGeolien** [comment]
Also on a similar note the typescript for certain functions seems vague: `getArrays(): any`

**Comment by daker** [comment]
Fixed in v25.8.1

@jourdain
